ZenPacks/zenoss/OpenStackInfrastructure/Instance.py

```diff
@@ -47,7 +47,7 @@ def set_host_name(self, hostname):
         if len(hypervisors):
             hypervisor = sorted(hypervisors)[0]
             if len(hypervisors) > 1:
-                LOG.error("Multiple hypervisors were found matching hostname %s - Choosing %s" %
+                LOG.warn("Multiple hypervisors were found matching hostname %s - Choosing %s" %
```

Comment:
I saw this happening quite often, and usually no bad things happen even if we have this 'error' msg. That's why I changed it to warning.